tests/unit/samplers/test_flowmc.py

```
@@ -0,0 +1,134 @@
+    s.sample(jax.random.key(1), jnp.ones((10, 2)) * 0.5)
+    result = s.get_samples()
+    # Samples are in sampling space = prior space (no transforms for this problem).
+    x = result["samples"][:, 0]
+    y = result["samples"][:, 1]
+    assert np.all(x >= 0.0) and np.all(x <= 1.0)
+    assert np.all(y >= 0.0) and np.all(y <= 1.0)
+
+
+def test_flowmc_sampler_get_samples_before_sample_raises():
+    s = _make_sampler()
+    with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)